kura/org.eclipse.kura.core.configuration/src/main/java/org/eclipse/kura/core/configuration/util/ComponentUtil.java

```diff
@@ -331,7 +332,7 @@ public static Tocd readObjectClassDefinition(Bundle bundle, String pid)
 
 				for (AD attrDef : attrDefs) {
 
-					String name = attrDef.getName();
+					String name = attrDef.getId();
```

Comment:
@ctron just to avoid confusion can you please rename the ``name`` local variable to ``id``?